sympy/utilities/lambdify.py

```diff
@@ -739,7 +738,14 @@ def _preprocess(self, args, expr):
                 argstrs.append(self._argrepr(dummy))
                 expr = self._subexpr(expr, {arg: dummy})
             else:
-                argstrs.append(str(arg))
+                argrep = self._argrepr(arg)
```

Comment:
Will this `argrep` be used somewhere?

Reply:
No it is not needed.
So I suppose one could do the following:
```diff
@@ -737,14 +737,11 @@ def _preprocess(self, args, expr):
                 dummy = Dummy()
                 argstrs.append(self._argrepr(dummy))
                 expr = self._subexpr(expr, {arg: dummy})
-            else:
-                argrep = self._argrepr(arg)
-
-                if dummify:
+            elif dummify:
                     dummy = Dummy()
                     argstrs.append(self._argrepr(dummy))
                     expr = self._subexpr(expr, {arg: dummy})
-                else:
+            else:
                     argstrs.append(str(arg))

```
Is this change appropriate ? If so, how do I proceed as this pull request has already been merged.

Reply:
The change looks good to me. Only, the level of indentation of the lines after `elif dummify:` and after `else:` should be reduced. You can open another PR .